documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+
+
+**Q:** Docker resources: We use an SSL listener, SSL port, and an administration port on each instance. How do we map these resources?
+
+**A:** Yes, you can use an SSL listener, SSL port, and an administration port in both Docker and Kubernetes.  See to the blog, [T3 RMI Communication for WebLogic Server Running on Kubernetes](https://blogs.oracle.com/weblogicserver/t3-rmi-communication-for-weblogic-server-running-on-kubernetes), which describes how to map these ports in Kubernetes.
+
+***
+
+#### Transactions
+
+**Q:** Are XA transactions and recovery also supported?
+
+**A:** Yes, XA transactions are supported. We are expanding our certification to include more complex cross-domain XA transaction use cases.
+
+***
+
+**Q:** For binding EJBs (presentation/business-tier), are unique and/or dynamic domain-names used?
+
+**A:** We do not enforce unique domain names. If you deploy two domains that must interoperate using RMI/EJB/JMS/JTA/and such, or that share RMI/EJB/JMS/JTA/and such clients, which concurrently communicate with both domains, then, as usual, the domain names must be configured to be different (even if they have different `domain-uids`).
+
+***
+
+#### Load Balancers
+
+**Q:** Load balancing and failover inside a DataCenter (HTTPS and T3s)?
+
+**A:** We originally certified on Traefik with the Kubernetes cluster; this is a very basic load balancer.  We have also certified other more sophisticated load balancers. See [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}).
+
+***
+
+#### Life Cycle and Scaling
+
+**Q:** How to deal with grow and shrink? Cluster and non-cluster mode.
+
+**A:** You can scale and shrink a configured WebLogic cluster (a set of preconfigured Managed Servers) or a dynamic WebLogic cluster (a cluster that uses templated Managed Servers) using different methods. See the blog, [Automatic Scaling of WebLogic Clusters on Kubernetes](https://blogs.oracle.com/weblogicserver/automatic-scaling-of-weblogic-clusters-on-kubernetes-v2).
+* Manually, using Kubernetes command-line interface, `kubectl`.
+* WLDF rules and policies; when the rule is met the Administration Server sends a REST call to the operator which calls the Kubernetes API to start a new pod/container/server.
+* We have developed and made open source the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) which exports WebLogic metrics to Prometheus and Grafana.  In Prometheus, you can set rules similar to WLDF and when these rules are met, a REST call is made to the operator which invokes a Kubernetes API to start a new pod. See to blog, [Announcing the New WebLogic Monitoring Exporter](https://blogs.oracle.com/weblogicserver/announcing-the-new-weblogic-monitoring-exporter-v2).
```

Comment:
Is this blog up-to-date? It's dated 2017.

Reply:
Doubt it. These FAQ are old as are their references. If the blog is no longer applicable, we should the delete the reference to it.